source/administration/mmctl-cli-tool.rst

```diff
@@ -719,139 +719,146 @@ mmctl channel
 Commands for channel management.
 
   Child Commands
-    -  `mmctl channel add`_ - Add a channel
     -  `mmctl channel archive`_ - Archive a channel
     -  `mmctl channel create`_ - Create a channel
+    -  `mmctl channel delete`_ - Delete a channel
     -  `mmctl channel list`_ - List all channels on specified teams
     -  `mmctl channel make_private`_ - Set a channel's type to "private"
     -  `mmctl channel modify`_ - Modify a channel's type (private/public)
-    -  `mmctl channel move`_ - Moves channels to the specified team
-    -  `mmctl channel remove`_ - Remove users from a channel
+    -  `mmctl channel move`_ - Move channels to the specified team
     -  `mmctl channel rename`_ - Rename a channel
-    -  `mmctl channel restore`_ - Restore a channel from the archive
+    -  `mmctl channel restore`_ - (Deprecated) Restore a channel from the archive
     -  `mmctl channel search`_ - Search a channel by name
+    -  `mmctl channel unarchive`_ - Unarchive a channel
+    -  `mmctl channel users`_ - Manage channel users
+    -  `mmctl channel users add`_ - Add a user to a channel
+    -  `mmctl channel users remove`_ - Remove a user from a channel
 
 **Options**
 
 .. code-block:: sh
 
    -h, --help   help for channel
 
-mmctl channel add
-^^^^^^^^^^^^^^^^^
+mmctl channel archive
+^^^^^^^^^^^^^^^^^^^^^
 
 **Description**
 
-  Add users to a channel. If adding multiple users, use a space-separated list.
+  Archive one or multiple channels along with all related information including posts from the database. Channels can be specified by ``[team]:[channel]`` (i.e., myteam:mychannel) or by channel ID.
```

Comment:
> with all related information including posts from the database. 

It is not clear to me what is happening in this part of the sentence. We don't actually delete from the database, so do we need to actually call out anything about it. Wondering if "Archive one or multiple channels along with all related posts data."  would be clearer?

Reply:
Good question! @mgdelacroix Would @wiersgallak's suggested update be clearer to the target audience from your perspective? I believe an update like this would apply to multiple child commands.

Reply:
As part of archiving a channel, we do things like deleting the related incoming and outgoing webhooks as well, so I would try keep the scope of the sentence broad, to explicit that we archive different types of information.

Reply:
@wiersgallak Are there additional updates you'd like to like to see made to this documentation?